docs/questions/query-builder/expressions/offset.md

```diff
@@ -20,6 +20,20 @@ Example: `Offset(Sum([Total]), -1)` would get the value of `Sum([Total])` from t
 
 Because `Offset` refers to other rows, the order of the breakouts matters (the breakouts are the groups in the "Group By" section in the Summarization step). Metabase will sort by the first group, then partition by any additional breakouts. For example, if you want to see the counts of orders by product category over time, and the counts by product category for the previous period, you should first group by `Created At`, then by the product category.
 
+## Offset doesn't account for missing data
+
+`Offset` can only reference the rows you _have_, which means it can produce correct but unexpected results. For example, lets say you want to compare each _day's_ count to the previous day.
+
+Take the following table, where the `Previous` column is created by the expression `Offset(Count, -1)`:
+
+| Date     | Count | Previous |
+| -------- | ----- | -------- |
+| 1-Oct-22 | 6     | 3        |
```

Comment:
i think it's better to either add a row of `...` before the first row to indocate that there's other data, or remove the Previous count for the first row. Otherwise it seems to contractict "`Offset` can only reference the rows you _have_" - this 3 comes from data you don't have.
Also, I think it should be changed to a different number because there's another 3 in the column which might be confusing